src/UI/Components/Diagram/Canvas.tsx

```diff
@@ -103,3 +128,43 @@ const Container = styled.div`
   margin: 0 40px;
   height: 100%;
 `;
+
+const ZoomWrapper = styled.div`
+  z-index: 10;
```

Comment:
From personal experience, I know this can become easily a mess when you have to add more elements in between or over or under that index. 
At my previous projects, we would have a global set of variables for the z-indexes. Which allowed to have an overview of the used index and easily adjust them if needed. I don't know if you like that approach or not.

Reply:
forgot to delete that, that was my attempt to "hide" scrollbar, but it didn't help and I ended up moving these buttons a little bit :D But I agree that z-indexes can get messy very fast